frontend/src/pages/Login.js

```diff
@@ -20,14 +27,16 @@ const Login = ({ setUser, successRedirection }) => {
       });
       if (!response?.data?.token) {
         console.log("Something went wrong during signing in: ", response);
+        localStorage.setItem("ERROR_MESSAGE", "SIGNIN_ERROR");
```

Comment:
to remove or replace by 

```suggestion
    setErrorMessage("SIGNIN_ERROR");
```